numpy/__init__.py

```diff
@@ -185,8 +185,12 @@ def pkgload(*packages, **options):
     pkgload.__doc__ = PackageLoader.__call__.__doc__
 
     from .testing import Tester
-    test = Tester().test
-    bench = Tester().bench
+    if '.dev0' in __version__:
+        test = Tester(raise_warnings="develop").test
+        bench = Tester(raise_warnings="develop").bench
+    else:
+        test = Tester(raise_warnings="release").test
+        bench = Tester(raise_warnings="release").bench
```

Comment:
This is `numpy/__init__.py`, so your sample code is missing several `del` statements that are necessary to avoid namespace pollution (and even without that, it's already longer than the code as written). I tried writing it a few ways, and this seemed the most readable overall.

AFAIK there's no reason for the `test` and `bench` object instances to be different; in fact AFAIK there's no reason for `bench` to exist at all, b/c I think we use asv these days? But here I just went for the minimal change to keep the old behavior...